components/Layout/layout.module.scss

```diff
@@ -2,12 +2,13 @@
 
 .layout {
   background-color: $techSwitchBlack;
-  width: 100%;
+  width: 100vw;
   height: 100vh;
   padding: 0;
   margin: 0;
   border-style: none;
   font-family: $techSwitchFont;
+  overflow: hidden;
```

Comment:
As discussed on slack - I think there's probably a better fix for this.
At the moment the issue is that there is some stuff that's not fitting on the page correctly.

Better if possible to fix that issue, rather than just cutting off the edge. :)